### ISSUE TITLE
[FEATURE]: Add to Playlist button

### ISSUE THREAD

**Issue by realityuk** [XeroxDev/YTMD-StreamDeck#50]
### Consent

- [X] I have searched the existing issues and verified that there is no open issue for the same subject.
- [X] I understand that the YTMDesktop Elgato Plugin has NO affiliation with Google, YouTube or the YTMDesktop application.

### Feature

An option to add a song to playlist, for instance if you enjoy a song on autoplay.

**Comment by XeroxDev** [comment]
Hey. Thank you for helping to improve the plugin. But before I add this feature, I've two questions: 
1. I don't have access to read out all playlists, but I have to provide the playlist id, or else it can't be added. How could this feature be added as user friendly as possible?
2. Would it also be enough to add the song to the library, instead of a playlist (this would solve the problem with 1.)?

**Comment by realityuk** [comment]
> 2. Would it also be enough to add the song to the library, instead of a playlist (this would solve the problem with 1.)?

Yeah, that's fine. Anything that works as user-friendly as possible would be awesome :)

**Comment by XeroxDev** [comment]
Okay, then I'll add those to the next release.

/assign @XeroxDev
/label "+In Progress"
/update title "[FEATURE]: Add to library action"

**Comment by BauaMomo** [comment]
>1. I don't have access to read out all playlists, but I have to provide the playlist id, or else it can't be added. How could this feature be added as user friendly as possible?

So on a Spotify plugin you can in the stream deck app choose which playlist the song should be added to. So you would have one button per playlist. Is this also not an option?

**Comment by XeroxDev** [comment]
Like mentioned above: I don't have access to read out all playlists. The only way this would be possible is to add an input field for the user to insert the playlist id and this is really user unfriendly and would IMO produce more problems due to users don't even know what an id is, where to get it and how to copy it.

**Comment by BauaMomo** [comment]
I see. Personally, I would rather have a button and input the ID manually than not have a button at all, but I can see how this could cause problems.
